docs/guides/LiveCode Builder Language Reference.md

```diff
@@ -703,6 +703,74 @@ on Android and iOS).
 > executed on arbitrary threads, as this is likely to cause your application
 > to crash.
 
+#### Foreign Code Libraries
+
```

Comment:
This all should be (some of it already is) in ide/docs/guides/Extending Livecode.md